repository-data/application/src/main/resources/hcm-config/namespaces/publicationsystem/attachment.yaml

```diff
@@ -0,0 +1,90 @@
+---
+definitions:
+  config:
+    /hippo:namespaces/publicationsystem/attachment:
+      /editor:templates:
+        /_default_:
+          jcr:primaryType: frontend:plugincluster
+          frontend:properties:
+          - mode
+          frontend:references:
+          - wicket.model
+          - model.compareTo
+          - engine
+          - validator.id
+          frontend:services:
+          - wicket.id
+          - validator.id
+          /root:
+            item: ${cluster.id}.field
+            jcr:primaryType: frontend:plugin
+            plugin.class: org.hippoecm.frontend.service.render.ListViewPlugin
+          /attachmentText:
+            /cluster.options:
+              jcr:primaryType: frontend:pluginconfig
+            caption: Attachment Text
```

Comment:
I'm not sure about this value. Unless the current value is what's been agreed with Martin, how about changing this to simply 'Name', 'Title', 'Custom Name', 'Display Name' or something similar?

It may be just me but 'Attachment Text' suggests a meaning close to 'Attachment Content' which may incur some pause on behalf of an author - and the effort of thinking to click the hint to get clarification of the meaning.

Reply:
Agreed, will update to Display Name